.gitignore

```diff
@@ -9,6 +9,7 @@
 .classpath
 .externalToolBuilders/
 .gradle/
+.vscode/
```

Comment:
Why? The horror. ;-)

Reply:
1) to annoy you
2) it doesn't use up all my ram like other IDEs - try it.  i was surprised too

Reply:
True on the ram. But for java/kotlin, you should use Intellij. Yeah, it uses ram, but it will help you learn Java a lot faster given it show hints for many things.